backend/src/middleware/notifications/notificationsMiddleware.spec.js

```diff
@@ -77,14 +77,18 @@ afterEach(async () => {
 describe('notifications', () => {
   const notificationQuery = gql`
     query($read: Boolean) {
-      currentUser {
-        notifications(read: $read, orderBy: createdAt_desc) {
-          read
-          reason
-          post {
+      notifications(read: $read, orderBy: createdAt_desc) {
+        read
+        reason
+        createdAt
+        from {
+          __typename
+          ... on Post {
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-30T07:51:46Z" title="Friday, August 30th 2019, 9:51:46 am +02:00">Aug 30, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -77,14 +77,15 @@ afterEach(async () => {
 describe('notifications', () => {
   const notificationQuery = gql`
     query($read: Boolean) {
-      currentUser {
-        notifications(read: $read, orderBy: createdAt_desc) {
-          read
-          reason
-          post {
+      notifications(read: $read, orderBy: createdAt_desc) {
+        read
+        reason
+        from {
+          __typename
+          ... on Post {
```


Cool stuff !!! 😍

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-30T12:01:31Z" title="Friday, August 30th 2019, 2:01:31 pm +02:00">Aug 30, 2019</time>_

---


it's like magic